spec/my_methods_spec.rb

```diff
@@ -0,0 +1,102 @@
+require_relative '../my_methods'
+
+RSpec.describe Enumerable do
+  let(:test_array) { [2, 2, 2, 4, 8] }
+
+  let(:test_array_1) { [1, 2, 3, 4] }
+
+  let(:test_array_2) { [2, 2, 2, 5, 2] }
+
+  describe '#my_each' do
+    it 'returns each of the elements in the array' do
+      expect(test_array_1.my_each { |element| element }).to eql(test_array_1.each { |element| element })
```

Comment:
Well done, like this you are testing against the methods you are trying to emulate :100: :white_check_mark: